closure/typedefs/AuditResult.js

```diff
@@ -74,8 +74,8 @@ AuditResult.prototype.extendedInfo;
 /** @type {string} */
 AuditResult.prototype.name;
 
-/** @type {!Array<string>} */
-AuditResult.prototype.tags;
+/** @type {string} */
+AuditResult.prototype.tag;
```

Comment:
`category`? Somewhat strange to have singular `tag` when the purpose of tags is usually to at least support being plural :)